atd-vze/src/views/Crashes/RelatedRecordsTable.js

```diff
@@ -37,14 +44,18 @@ const RelatedRecordsTable = ({
   };
 
   const handleInputChange = (e, updateFieldKey) => {
+    // Change new value to null if it only contains whitespaces
+    // or save value as all uppercase
+    const newValue =
+      e.target.value.trim().length === 0 ? null : e.target.value.toUpperCase();
```

Comment:
this handles the input for victim names but also works for the other text input types